### ISSUE TITLE
Epub validation error - Error while parsing file 'element "div" 

### ISSUE THREAD

**Issue by jamtur01** [jgm/pandoc#4087]
Not sure if I should be logging this here or the skylighting repo.

I create an ePub with 2.0.3 and when I validate it I see numerous validation errors like:

```
ERROR(RSC-005): /Users/james/src/pm/output/Title.epub/EPUB/text/ch005.xhtml(374,58): Error while parsing file 'element "div" not allowed here; expected the element end-tag, text or element "a", "abbr", "area", "audio", "b", "bdi", "bdo", "br", "button", "canvas", "cite", "code", "command", "datalist", "del", "dfn", "em", "embed", "epub:switch", "i", "iframe", "img", "input", "ins", "kbd", "keygen", "label", "link", "map", "mark", "meta", "meter", "ns1:math", "ns2:svg", "object", "output", "progress", "q", "ruby", "s", "samp", "script", "select", "small", "span", "strong", "sub", "sup", "textarea", "time", "u", "var", "video" or "wbr" (with xmlns:ns1="http://www.w3.org/1998/Math/MathML" xmlns:ns2="http://www.w3.org/2000/svg")'.
```

The HTML on 374 and column 58 is the space between the `<div>` and the first `<span>` on the last line:

```html
<pre class="sourceCode bash" id="cb24" title="Checking the Alertmanager version on Windows"><code class="sourceCode bash"><div class="sourceLine" id="cb24-1" data-line-number="1"><span class="ex">C</span>:<span class="dt">\&gt;</span> alertmanager.exe -version</div>
<div class="sourceLine" id="cb24-2" data-line-number="2"><span class="ex">alertmanager</span>, version 0.11.0 (branch: HEAD, revision: 30dd0426c08b6479d9a26259ea5efd63bc1ee273)</div>
<div class="sourceLine" id="cb24-3" data-line-number="3">  <span class="ex">build</span> user:       root@3e103e3fc918</div>
<div class="sourceLine" id="cb24-4" data-line-number="4">  <span class="ex">build</span> date:       20171116-17:45:26</div>
<div class="sourceLine" id="cb24-5" data-line-number="5">  <span class="ex">go</span> version:       go1.9.2</div></code></pre>
```

**Comment by jgm** [comment]
Ah, blast. Yes, the code element is supposed to have
"phrasing content" inside it, and div is excluded.
Some further changes in the skylighting library will
be needed.

+++ James Turnbull [Nov 22 17 03:49 ]:

**Comment by jgm** [comment]
This is fixed by compiling with skylighting 0.4.4 or higher.
